Collections/Bag/Bag.swift

```diff
@@ -4,6 +4,12 @@
 
 import Foundation
 
+// MARK: - Subtypes
+enum RemovalType {
+    case any(Int)
+    case all
+}
+
```

Comment:
Please include this as a private subtype to the Bag struct, not top-level

Reply:
Of course this is not possible, because an internal method cannot have a private type as a parameter.